share/tools/create_manpage_completions.py

```diff
@@ -774,7 +774,8 @@ def parse_manpage_at_path(manpage_path, output_directory):
 
             # Output the magic word Autogenerated so we can tell if we can overwrite this
             built_command_output.insert(1, "# Autogenerated from man page " + manpage_path)
-            built_command_output.insert(2, "# using " + parser.__class__.__name__)
+            # built_command_output.insert(2, "# using " + parser.__class__.__name__) # XXX MISATTRIBUTES THE CULPABILE PARSER! Was really using Type2 but reporting TypeDeroffManParser
```

Comment:
```
Considering /usr/share/man/man1/xcode-select.1
   Trying Type2ManParser
      Command is u'xcode-select'
   Trying Type3ManParser
      Command is u'xcode-select'
      Unable to find options section
   Trying TypeDarwinManParser
   Trying TypeDeroffManParser
   /usr/share/man/man1/xcode-select.1 parsed successfully
```
It was reporting the last parser that was used which ends up not being the same one that actually did the work. This was super annoying!